lib/views/home.dart

```diff
@@ -1,10 +1,189 @@
+// ignore_for_file: prefer_const_constructors
+
 import 'package:flutter/material.dart';
 import 'package:get/get_state_manager/get_state_manager.dart';
 import 'package:thirumathikart_app/controllers/home_controller.dart';
+import 'package:thirumathikart_app/config/themes.dart';
+// ignore: import_of_legacy_library_into_null_safe
```

Comment:
Same here remove commented lines( the library version is not supporting null safety ,change version)